code/modules/clothing/suits/bio.dm

```diff
@@ -69,20 +69,20 @@
 /obj/item/clothing/suit/bio_suit/scientist
 	icon_state = "bio_scientist"
 
-
 //CMO's biosuit, blue stripe
-/obj/item/clothing/suit/bio_suit/cmo
-	icon_state = "bio_cmo"
-
 /obj/item/clothing/head/bio_hood/cmo
 	icon_state = "bio_cmo"
 
+/obj/item/clothing/suit/bio_suit/cmo
+	icon_state = "bio_cmo"
+	allowed = list(/obj/item/tank/internals, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/glass/beaker, /obj/item/melee/classic_baton/telescopic)

```

Comment:
let it inher from base then add telescopic in initialize? that should be the only difference tbh.

Reply:
done